go-controller/.golangci.yml

```diff
@@ -4,6 +4,7 @@ run:
     - pkg/crd
 
   skip-files:
+    # TODO: lint test files as well
```

Comment:
the unused parameter commit looks good to me, however the only question I have was unit tests files and looks like we are skipping it :D -> ok that explains why I see unused params in unit test files still